chainerx_cc/chainerx/native/native_device/pool.cc

```diff
@@ -24,6 +25,7 @@
 #include "chainerx/native/kernel_regist.h"
 #include "chainerx/native/tensor_dot.h"
 #include "chainerx/numeric_limits.h"
+#include "chainerx/routines/arithmetic.h"
```

Comment:
I guess `kernels/arithmetic.h` is for `Divide(AS)Kernsl`, but do you need `routines/arithmetic.h`?